api/resolvers/blockHeight.js

```diff
@@ -0,0 +1,35 @@
+const cache = new Map()
+const expiresIn = 1000 * 60 // in milliseconds
```

Comment:
lol, I tried to find out the rate limits for the mempool API but only found this

> Our API limits aren't disclosed, basically because if you have to ask then you will hit them. For this type of application it's best if you run your own node and query it locally.

-- https://github.com/mempool/mempool/discussions/752#discussioncomment-1248907

Now I am confused. Is 1 minute too often? lol

Reply:
That's... an interesting choice.

Regardless, I'll refactor to use LND instead.